<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>59</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>83</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>41</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;PHONEBOOK&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>82</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>59</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>117</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="MainADDBUTTON">
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MainSEARCHBUTTON">
        <property name="text">
         <string>SEARCH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MainREMOVEBUTTON">
        <property name="text">
         <string>REMOVE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MainSAVEBUTTON">
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveLOADBUTTON">
        <property name="text">
         <string>LOAD</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>115</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3" rowspan="2">
     <widget class="QPushButton" name="MainEXIT">
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>237</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>389</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textBrowser</tabstop>
  <tabstop>MainADDBUTTON</tabstop>
  <tabstop>MainSEARCHBUTTON</tabstop>
  <tabstop>MainREMOVEBUTTON</tabstop>
  <tabstop>MainSAVEBUTTON</tabstop>
  <tabstop>SaveLOADBUTTON</tabstop>
  <tabstop>MainEXIT</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
